docs/abcd_rooparametrichist_tutorial/rooparametrichist_exercise.md

```diff
@@ -0,0 +1,507 @@
+After running the script, the workspace will be saved in ```example_analysis/datacards/```. To create the datacards automatically fatching the correct workspace, run:
+
+
+```
+python3 utils/create_datacards.py -m 1500
+
+```
+
+The datacards can be combined then using the usual command:
+
+```
+combineCards.py mPhi_1500_*2018*.txt > combined_mPhi_1500_2018.txt
+
+```
+
+<details>
+<summary> Datacard with A,B,C,D regions combined  </summary>
+
+```
+Combination of mPhi1500_Catany_2018_CR_B.txt  mPhi1500_Catany_2018_CR_C.txt  mPhi1500_Catany_2018_CR_D.txt  mPhi1500_Catany_2018_SR.txt
+imax 4 number of bins
+jmax 1 number of processes minus 1
+kmax 2 number of nuisance parameters
+----------------------------------------------------------------------------------------------------------------------------------
+shapes Bkg       ch1       param_ws.root wspace:bkg_B
+shapes data_obs  ch1       param_ws.root wspace:data_obs_B
+shapes mPhi_1500  ch1      param_ws.root wspace:mPhi_1500_B
+shapes Bkg       ch2       param_ws.root wspace:bkg_C
+shapes data_obs  ch2       param_ws.root wspace:data_obs_C
+shapes mPhi_1500  ch2      param_ws.root wspace:mPhi_1500_C
+shapes Bkg       ch3       param_ws.root wspace:bkg_D
+shapes data_obs  ch3       param_ws.root wspace:data_obs_D
+shapes mPhi_1500  ch3      param_ws.root wspace:mPhi_1500_D
+shapes Bkg       ch4       param_ws.root wspace:bkg_A
+shapes data_obs  ch4       param_ws.root wspace:data_obs_A
+shapes mPhi_1500  ch4      param_ws.root wspace:mPhi_1500_A
+----------------------------------------------------------------------------------------------------------------------------------
+bin          ch1    ch2    ch3    ch4  
+observation  -1     -1     -1     -1   
+----------------------------------------------------------------------------------------------------------------------------------
+bin                             ch1       ch1       ch2       ch2       ch3       ch3       ch4       ch4     
+process                         mPhi_1500  Bkg      mPhi_1500  Bkg      mPhi_1500  Bkg      mPhi_1500  Bkg     
+process                         0         1         0         1         0         1         0         1       
+rate                            -1        1         -1        1         -1        1         -1        1       
+----------------------------------------------------------------------------------------------------------------------------------
+BkgRate                 lnN     -         -         -         -         -         -         -         1.05    
+lumi                    lnN     1.016     -         1.016     -         1.016     -         1.016     -
+
+
+```
+</details>
+
+
+
+## Run Fit
+<a id="fit"></a>
+
+From the combined datacard for all the regions, one can run the usual fit diagnostics as follows:
+
+```
+combine -M FitDiagnostics combined_mPhi_1500_2018.txt -m 1500 --saveShapes --saveWithUncertainties --saveNormalizations
```

Comment:
Here, I am now getting an issue that the datacard seems to be looking for `mPhi_{mass}_{region}` but the relevant histograms in my workspace appear to be saved as `mPhi{mass}_{region}`.

Reply:
this should be solved now since i forgot to push the new version of the script creating the workspace